format_docs.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+prettier -w src
```

Comment:
I dont think this script is really useful. More importantly there are no installation instructions. So I would really move it to the readme with commands for install (Ubuntu) + usage.